spring-ws-core/src/main/java/org/springframework/ws/server/endpoint/observation/DefaultWebServiceEndpointConvention.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2005-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.ws.server.endpoint.observation;
+
+import io.micrometer.common.KeyValue;
+import io.micrometer.common.KeyValues;
+import io.micrometer.observation.Observation;
+
+/**
+ * Default ObservationConvention for a WebService Endpoint.
+ * @author Johan Kindgren
+ */
+public class DefaultWebServiceEndpointConvention implements WebServiceEndpointConvention {
+
+    private static final KeyValue EXCEPTION_NONE = KeyValue.of(EndpointObservationDocumentation.LowCardinalityKeyNames.EXCEPTION, KeyValue.NONE_VALUE);
+    private String name = "webservice.server";
+
+
+    @Override
+    public KeyValues getLowCardinalityKeyValues(WebServiceEndpointContext context) {
+        return KeyValues.of(
+                exception(context),
```

Comment:
Values can be null?

Reply:
All of the keys are either "unknown" or "none" if no value is present.